src/modality-specific-files/physiological-recordings.md

```diff
@@ -72,122 +194,585 @@ A guide for using macros can be found at
+        "Units": "a.u."
+    }
+}
+```
+
+Content of `sub-01_task-VisualSearch_events.json`:
+
+```JSON
+{
+   "TaskName": "Visual Search",
+   "InstitutionName": "Stanford University",
+   "InstitutionAddress": "450 Serra Mall, Stanford, CA 94305-2004, USA",
+   "StimulusPresentation": {
+       "ScreenDistance": 60,
+       "ScreenRefreshRate": 60,
+       "ScreenResolution": [1024, 768],
+       "ScreenSize": [0.386, 0.29]
    }
-) }}
+}
+```
+
+**Example eye-tracking recording**.
+Given the above `sub-01_task-visualSearch_recording-eye1_physio.json` metadata
+specification, the decompressed content of the
+`sub-01_task-visualSearch_recording-eye1_physio.tsv.gz` can be:
+
+```TSV
+7186799    416.29    267.39    4612.0
+7186800    416.29    268.10    4623.0
+7186801    416.20    269.00    4623.0
+7186802    415.89    269.60    4613.0
+7186803    415.70    269.20    4603.0
+7186804    415.60    266.79    4591.0
+7186805    415.79    264.60    4589.0
+7186806    416.10    263.89    4587.0
+7186807    416.29    265.20    4587.0
+7186808    416.39    266.50    4588.0
+7186809    416.50    266.79    4594.0
+7186810    416.50    267.20    4599.0
+7186811    416.10    268.00    4609.0
+7186812    415.70    268.29    4612.0
+7186813    416.00    268.60    4605.0
+```
+
+Example `sub-01_task-visualSearch_recording-eye1_physioevents.tsv.gz` corresponding
+to the above eye-tracking recording, after decompressing:
+
+```TSV
+7184392    n/a    n/a         n/a    "NO Reply is disabled for function eyelink_cal_result"
+7184392    n/a    n/a         n/a    "RECCFG CR 1000 2 0 R"
+7184392    n/a    n/a         n/a    "ELCLCFG TOWER"
+7186771    n/a    n/a         n/a    "First task trigger"
+7186806    72     fixation    0      n/a
+7186879    231    saccade     1      n/a
+7187111    6186   fixation    0      n/a
+7193298    216    saccade     1      n/a
+7193515    1286   fixation    0      n/a
+7194802    24     saccade     0      n/a
+7194827    2403   fixation    0      n/a
+7197231    17     saccade     0      n/a
+7197249    1640   fixation    0      n/a
+7198890    6      saccade     0      n/a
+7198897    1105   fixation    0      n/a
+7200003    233    saccade     1      n/a
+7200237    184    fixation    0      n/a
+7200422    15     saccade     0      n/a
+7200438    264    fixation    0      n/a
+```
+
+where the first three rows are logged by the eye-tracker and the fourth row shows
+a *message* asynchronously received by the eye-tracker.
+The remainder of the rows contain a subset of parameters registered by the device,
+derived from applying a model to identify saccades and blinks.
+The corresponding `sub-01_task-visualSearch_recording-eye1_physioevents.json` sidecar
+would read:
+
+```JSON
+{
+    "Columns": ["onset", "duration", "trial_type", "blink", "message"],
+    "Description": "Messages logged by the measurement device",
+    "ForeignIndexColumn": "timestamp",
+    "blink": {
```

Comment:
shameless plug: this is the situation I would like to avoid in
- https://github.com/bids-standard/bids-2-devel/issues/71

where I guess such file would look like

```json
{
    "Description": "Messages logged by the measurement device",
    "ForeignIndexColumn": "timestamp",
    "Columns": {
       "onset": { "Index": 1},
       "duration": {"Index": 2},
       "trial_type": {
           "Description": "Event type as identified by the eye-tracker's model (either 'n/a' if not applicabble, 'fixation', or 'saccade')."
           "Index": 3}, 
       "blink": {
         "Description": "One indicates if the eye was closed, zero if open."
         "Index" : 4},
       "message": {
         "Description": "String messages logged by the eye-tracker.",
         "Index": 5}
}
```

Reply:
I'm +1 on this and the solution.

However, I understand that can only be implemented in BIDS 2.0 because would introduce backward breaking changes, right?

If so, I think this is something we cannot resolve here :(